go2ts.go

```diff
@@ -43,6 +43,18 @@ func (g *Go2TS) Add(v interface{}) error {
 	return g.AddWithName(v, "")
 }
 
+// AddMultiple types in a single call.
```

Comment:
// AddMultiple **adds multiple** types in a single call.

Reply:
Done.